app/models/concerns/curation_concerns/collection.rb

```diff
@@ -10,5 +10,17 @@ def add_members(new_member_ids)
       return if new_member_ids.nil? || new_member_ids.empty?
       members << ActiveFedora::Base.find(new_member_ids)
     end
+
+    def add_member_objects(new_member_ids)
```

Comment:
Add some documentation here about how this adds the association on the member side. If you wouldn't mind, please also add some documentation to `add_members` too.